### ISSUE TITLE
Standardize logo filenames?

### ISSUE THREAD

**Issue by jbednar** [pyviz/nbsite#45]
Can we agree on a convention for naming logo files so that all projects can call the files by standard names to avoid me getting hopelessly confused?  The logo files from our graphic designer differ in various ways:

- whether they include an icon, word mark, or both
- when including both icon and word mark, whether they are horizontally aligned or vertically stacked
- how many colors they use (full RGB, flat, black&white)

I think we only want the full-color RGB files in our websites, so the relevant reference files for datashader are:

```
datashader-icon-full-color.png
datashader-stacked-full-color-RGB.png
datashader-horizontal-full-color-RGB.png
datashader-word-mark-color.png
```

Given that we only need those four, the names from the designer seem bulky and awkward, so I propose we call them something standardized but shorter, such as:

```
datashader_logo.png
datashader_logo_vert.png
datashader_logo_horiz.png
datashader_wm.png
```

Names like these should work for the sites like pyviz.org that need to refer to many different logos by name.  But for the ones forming the site itself, maybe we can just name them the same names in every project, so that they don't need updating in conf.py at all?

```
logo.png
logo_vert.png
logo_horiz.png
wm.png
```

I think we also need a medium sized version (between favicon.ico and logo.png), which could be `logo_medium.png` maybe?

**Comment by jlstevens** [comment]
Do you not want to say whether the colors are flat or not? I don't think we'll have much use for black and white logos but the holoplot logo (for instance) is flat while the geo/holoviews ones are not.

Otherwise, this proposal seems fine to me.

**Comment by jlstevens** [comment]
And you also have the difference between vector and raster but the file extension can handle that (.png vs .svg).

**Comment by jbednar** [comment]
Right, where I say "png" above, that should be read as shorthand for "png" or "svg".  I don't personally have any reason to distinguish flat from full-color logos in any of our work; do you?  A company needs to, if they are printing stickers or T-shirts, but I don't see why it would matter for notebooks, websites, and the like.  So I'm happy for any project's logo to be flat or full color as it prefers; whatever it is would be found at those filenames.

**Comment by jlstevens** [comment]
Sure. I guess that is fine if we never want to mix the flat and gradient versions.

**Comment by jbednar** [comment]
Right; can't see why we would.  I guess if it ever comes up, we can add _flat versions otherwise named as above.  The above names would be for the richest version of the logo available, i.e. full color with gradients for those projects where that's available, and otherwise the flat version.

Presumably we can use the same mechanism as in https://github.com/pyviz/nbsite/issues/40 to generate logo_stacked_m.png out of logo_stacked.png for use on the homepages or in the left-hand sidebar.
